apps/datafeeder/src/app/presentation/components/upload-data-rules/upload-data-rules.component.html

```diff
@@ -1,14 +1,15 @@
 <div class="text-orange-500">
   <div class="w-20 h-1 bg-orange-500 mb-4"></div>
-  <span class="font-bold">Accepted file formats:</span>
-  <div class="w-64 flex justify-between">
-    <span>- SHP</span>
-    <span>- GeoJSON</span>
+  <span class="font-bold" translate="">{{ 'Accepted file formats' }}:</span>
```

Comment:
Use meaningfull translation key `datafeeder.upload.acceptedFormats` instead of literal strings

Reply:
When you have ":" should ":" be part of the translation ? some language add a space before the ":" some others don't.